packages/components/src/index.tsx

```diff
@@ -0,0 +1,31 @@
+import TitleCell from './components/cells/title';
+import MarkdownCell from './components/cells/markdown';
+import CodeCell from './components/cells/code';
+export { TitleCell, MarkdownCell, CodeCell };
+
+export * from './components/ui/button';
+export * from './components/ui/card';
+export * from './components/ui/collapsible';
+export * from './components/ui/command';
+export * from './components/ui/dialog';
+export * from './components/ui/dropdown-menu';
+export * from './components/ui/heading';
+export * from './components/ui/input';
+export * from './components/ui/navigation-menu';
+export * from './components/ui/popover';
+export * from './components/ui/resizable';
+export * from './components/ui/select';
+export * from './components/ui/sheet';
+export * from './components/ui/sonner';
+export * from './components/ui/switch';
+export * from './components/ui/tabs';
+export * from './components/ui/textarea';
+export * from './components/ui/tooltip';
+export {
+  Tabs as UnderlineTabs,
+  TabsList as UnderlingTabsList,
+  TabsTrigger as UnderlineTabsTrigger,
+  TabsContent as UnderlineTabsContent,
+} from './components/ui/underline-flat-tabs';
+
+export * from './lib/utils';
```

Comment:
I added a root `src/index.tsx` reexporting everything in `@srcbook/components`. I suspect this is probably not something that anyone would want to use because it makes bundle splitting more more difficult, but on the other hand it seemed like there should be _something_ importable at the package root.